### ISSUE TITLE
GM clues requirements

### ISSUE THREAD

**Issue by RubenJS95** [oldschoolgg/oldschoolbot#1866]
## 🐛 Bug Report

To be able to complete GM clues you need to have completed an x amount of master clues. The game calculates this using your amount of opened master caskets.

However, when you have 0 opened master caskets, the game still allows you to complete GM clues. The system only seems to work when you have at least 1 opened master casket, but less than the required amount. As it then indeed unables you to complete GM clues.

[note] found this out prior to the newly required 100kc. This was still the case when GMs required 50kc (6-7 hours ago). So the issue could be resolved, but I can’t find that out anymore.

Tldr; minimum master clues completed in order to complete GMs doesn’t work when master completions is at 0.

**Comment by themrrobert** [comment]
Fixed in [1867](https://github.com/oldschoolgg/oldschoolbot/pull/1867)
